### DISCUSSION TITLE
about the JS

### DISCUSSION THREAD

**Discussion by VarshaKATE** [Jay-study-nildana/FrontEndForStudents#48] [Coding Questions]
Hi Jay,
this is varsha kate i am learning full stack web developer program, but as i am from civil background i am finding difficulty in javascript please help me get better understanding in that, as you said i have started practicing 4hr/day hoping positive responce from you.

thank you.

**Comment by Jay-study-nildana** [comment]
@VarshaKATE 

Hello and welcome to my learning repository. 4 hours per day, very good. I have put some useful information for new learners, in the main, readme page. Look at the section called "Useful Resources/Links/Repos and Other Stuff".

1. https://github.com/Jay-study-nildana/FrontEndForStudents/blob/main/README.md

and of course, you can post questions here on the discussion forum, and I will reply whenever I get time. 

You can connect with me and get in touch with me, using the following details. 

1. https://jay-study-nildana.github.io/developerprofile/
1. https://thechalakas.com

If you live in Bangalore/Mysore, feel free to drop by my office for free classes and career guidance. All the best for your learning. Happy Coding. 

Jay (Vijayasimha BR)

**Comment by VarshaKATE** [comment]
Thank you for your kind response sir,  unfortunately I m leaving in pune.
I will definitely follow you steps.

**Reply by Jay-study-nildana** [reply]
@VarshaKATE 

You are most welcome :) 

Jay (Vijayasimha BR)
